### PR TITLE
Editorial updates in the initial section of the README.

### PR DESCRIPTION
**Pull request by kbrandl**
This PR updates the initial section of the README to streamline wording and add links to the docs that describe how to use the Yeoman generator to create various types of add-ins.

### CODE REVIEW

**Comment by kbrandl** [comment]
@akrantz if these updates to the README look okay to you, please approve and merge. Thanks!

**Comment by akrantz** [comment]
@kbrandl Let me know if you're done making changes and I will merge.

File: readme.md [hunk 1]
```diff
@@ -34,7 +27,22 @@ Like other Yeoman generators, this simply creates the scaffolding of files for y
...
+- [Build an Outlook add-in](https://docs.microsoft.com/en-us/outlook/add-ins/quick-start?context=office%2Fdev%2Fadd-ins%2Fcontext)
+- [Build a PowerPoint add-in](https://docs.microsoft.com/en-us/office/dev/add-ins/quickstarts/powerpoint-quickstart)
+- [Build a Project add-in](https://docs.microsoft.com/en-us/office/dev/add-ins/quickstarts/project-quickstart)
+- [Build a Word add-in](https://docs.microsoft.com/en-us/office/dev/add-ins/quickstarts/word-quickstart)
+
+> **Note:** While you can also use [Visual Studio](https://visualstudio.microsoft.com/vs) to create Office Add-in projects, the Yeoman generator provides more options in terms of the types of projects you can create. For example:
+> - The Yeoman generator can create Office Add-in projects that use plain HTML, CSS & JavaScript, or React, or Angular, whereas Visual Studio can only create Office Add-ins projects that use plain HTML, CSS & JavaScript.
+> - The Yeoman generator can create 7 types of add-ins (Excel, OneNote, Outlook, PowerPoint, Project, Word, Excel Custom Functions), whereas Visual Studio can only create 4 types of add-ins (Excel, Outlook, PowerPoint, and Word).
```
[the remark this answers is not in the document]
Comment (kbrandl) [reply readme.md]: @akrantz that's a fair point. I've removed the links to individual docs and replaced with a general pointer to the Office Add-ins docs landing page instead. Given that this information at the top of the README is intended to be a general/high-level introduction to the Yeoman generator, seems like a matrix of 48 possibilities may not be necessary; support for those 48 possibilities is implied by the fact that we're already describing support for 4 project types, 6 Office apps, 2 language choices.

**Comment by kbrandl** [comment]
@akrantz I'm done now -- please feel free to merge. Thank you!

**Pull request merged**